CHANGES.md

```diff
@@ -4,6 +4,8 @@ wavesurfer.js changelog
 Next (unreleased)
 -----------------
 
+- Always call drawBuffer when for media-elements when peaks are provided, fixes
+  a missed call uncovered by #1969 (#1990)
```

Comment:
typo in sentence ("when for"?) and use backticks for `drawBuffer`.

Reply:
since #1969 hasn't been released yet you can leave out this changelog entry completely (and perhaps add reference to pull request instead to changelog entry of #1969).